tests-ui/platform/assets/components/AssetBrowserModal.test.ts

```diff
@@ -86,6 +91,19 @@ vi.mock('@/components/widget/panel/LeftSidePanel.vue', () => ({
   }
 }))
 
+vi.mock('@/platform/assets/components/AssetFilterBar.vue', () => ({
```

Comment:
Mocking the whole component, I like it 😁

Reply:
haha i just need to test the interface!